tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -124,6 +124,6 @@ def validate_section(self, section):
         """Validate the section field."""
         if DataFile.Section.is_fra(section):
             user = self.context.get('user')
-            if not user.has_fra_access:
+            if not user.has_fra_access and not user.is_ofa_sys_admin:
```

Comment:
i get this error when submitting an FRA file as an admin and the first row is blank (and other rows not blank). just want to confirm that this the expected behavior?